apache-maven/src/assembly/maven/conf/settings.xml

```diff
@@ -244,6 +244,35 @@ under the License.
       </properties>
     </profile>
     -->
+    <profile>
+      <activation>
```

Comment:
To me we're now abusing a profile because it is not yet possible to specify these at a higher level.  The `default` profile could contain much more.

Reply:
That's correct, but we don't have a better way. Settings with Maven 5 could natively support repositories. Do you see a better way?

Reply:
To me the `settings.xml` is a Maven specific file (unlike the `pom.xml`). I see this as an opportunity to improve the `settings.xml` right now. I don't like the idea of a temporary solution now of which we know it'll be changed. Those temporary hybrid style settings will be there forever.

Reply:
So you want to bump settings model to 2.0.0 and introduce repos there?

Reply:
version 1.2.0 would be good enough, we're only adding stuff, not breaking things.
And we're talking about `config/settings.xml`, not `.m2./settings.xml`. It must be possible to keep the latter at 1.0.0 or 1.1.0, while improving the first one.

Reply:
Correct, makes sense.

Reply:
So do you want to move this after settings 1.2.0 or can we start with a profile and then move to a separate repo? Implementing 1.2.0 will take some time...

Reply:
https://issues.apache.org/jira/browse/MNG-7059